packages/@lightningjs/ui-components/src/components/MetadataCardContent/MetadataCardContent.js

```diff
@@ -233,6 +233,7 @@ export default class MetadataCardContent extends MetadataBase {
 
     this._Provider.x = this._DetailsWrapper.w - this._providerW;
     this._Provider.y = this._DetailsWrapper.h - this._providerH;
+    this._updateDetails();
```

Comment:
This feels weird to have at the end of `_updatePositions`. Can you try moving this to the end of `_resolveProvider` and see if this still does the trick? I tested and it seems like it might work. That will make it so that it updates that portion of the component after the Provider is done loading, which I think is more likely what we want instead of it running every time we need to update positions (which gets called by several other update functions).

Reply:
good callout!

Reply:
I'll take a look to make sure it's working as expected. Thanks @ImCoolNowRight

Reply:
Looks good. Made the update.